src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -275,6 +276,6 @@ docker run --name $docker_name \
   --label PAI_USER_NAME={{{ jobData.userName }}} \
   --label PAI_CURRENT_TASK_ROLE_NAME={{{ taskData.name }}} \
   --env-file $ENV_LIST \
-  --entrypoint '/bin/bash' {{{ jobData.image }}} \
-  '/pai/bootstrap/docker_bootstrap.sh'
+   {{{ jobData.image }}} \
+  /bin/bash '/pai/bootstrap/docker_bootstrap.sh'
```

Comment:
this will set the pid of "bash docker_bootstrap.sh" to 1? it seems previously it is already the case.

Reply:
Not actually:
Without `--init`, pid 1 process is `bash docker_bootstrap.sh`.
With `--init`, pid 1 process is `/dev/init`, and `bash docker_bootstrap.sh` will be its subprocess.

Reply:
yeah. https://docs.docker.com/engine/reference/run/#specify-an-init-process

Reply:
In my testing, `--init` can fix the suicide failure as well.  it is a good solution to such issues.

https://stackoverflow.com/questions/49162358/docker-init-zombies-why-does-it-matter
https://github.com/krallin/tini/issues/8#issuecomment-146135930

Reply:
https://github.com/krallin/tini#understanding-tini. if the child of process exits, init will exit